### ISSUE TITLE
The inference class is different from the ade20k

### ISSUE THREAD

**Issue by ivanonair** [ayoolaolafenwa/PixelLib#9]
Thanks for sharing this module.
I want to use the pre-trained model to segment the floor from indoor scene pictures.
But I find the output array didn't include the floor label.
I can't get the same result with the pic on the medium post.
https://miro.medium.com/max/700/1*lh2Ud7_9XxwP3OKYiWX9GQ.jpeg
How could I turn on the 150 classes of ade20k dataset?
Thanks.

**Comment by ivanonair** [comment]
Oh sorry, I found the problem is the color setting.
Thanks for sharing such a great feature.

**Comment by VinayChaudhari1996** [comment]
> Oh sorry, I found the problem is the color setting.
> Thanks for sharing such a great feature.

Hey , 

Model : Using ade20k pretrained

I'm trying to get floor segmentation array and wants to change the floor with new image. 

How can I achieve that ? 

Thanks ,
